ocs_ci/ocs/cluster.py

```diff
@@ -82,6 +86,13 @@ def __init__(self):
             logging.warning("No CephFS found")
             self.cephfs_config = None
 
+        try:
+            self.rbd_config = self.RBD.get().get("items")[0]
+        except IndexError as e:
+            logging.warning(e)
+            logging.warning("No RBD found")
```

Comment:
@vasukulkarni we might hit this exception in case the cluster was deployed without RBD, and in that case we will not run tests against RBD, so it will not affect the creation of new RBD pool later on.